### ISSUE TITLE
Indexing google drive to plex library

### ISSUE THREAD

**Issue by jaredneedell** [astrada/google-drive-ocamlfuse#265]
I'm indexing a google drive folder to Plex and after a while the system can't read the folders anymore.

If I try to read the directory I'll get this error
`ls: cannot open directory .: Numerical result out of range`

I enabled debugging to see what the logs would say and this is what I see - 
`[24157.470062] TID=162525: Retrying (1/8) final_step after exception: Failure("Sqlite3 error: FULL")`

Have you seen this before?

**Comment by jaredneedell** [comment]
I should note if I clear the cache using -cc, I can get the folder to remount

**Comment by astrada** [comment]
The sqlite error means that your drive is full. With -debug every byte exchanged with Google gets logged. Try with -verbose.

**Comment by jaredneedell** [comment]
Is it possible to force ocaml to use a different folder for the cache? I have a mount point that has 500GB of free space which is more forgiving than the 10GB it has right now working out of the home folder.

**Comment by astrada** [comment]
Yes, you can choose a custom folder setting `cache_directory=` in the configuration file.

**Comment by pisto** [comment]
@jaredneedell you should probably look into force streaming of large files, that's what I do.

**Comment by jaredneedell** [comment]
Thanks @astrada I'll give it a try

@pisto 
I have that setting set but doesn't seem to do anything.

`stream_large_files=true`

**Comment by jaredneedell** [comment]
@astrada I changed the cache dir, it never got over 1mb and I was syncing my music folder to Plex when I got the error endpoint not connected. Took about 20 minutes.

**Comment by astrada** [comment]
Can you please send the log files to alessandro.strada@gmail.com? Thanks!
